### PR TITLE
Add an option to include call name in BQ table.

### PR DESCRIPTION
**Pull request by tneymanov**

### CODE REVIEW

File: gcp_variant_transforms/libs/schema_converter.py [hunk 1]
```diff
@@ -133,6 +135,13 @@ def generate_schema_from_header_fields(
       description='Unique ID (type INT64) assigned to each sample. Table with '
                   '`__sample_info` suffix contains the mapping of sample names '
                   '(as read from VCF header) to these assigned IDs.'))
+  if include_call_name:
+    calls_record.fields.append(bigquery.TableFieldSchema(
+        name=bigquery_util.ColumnKeyConstants.CALLS_NAME,
+        type=bigquery_util.TableFieldConstants.TYPE_STRING,
+        mode=bigquery_util.TableFieldConstants.MODE_NULLABLE,
+        description='Name of the call.'))
```
Comment (samanvp) [inline gcp_variant_transforms/libs/schema_converter.py:143]: How about this:
'Name of the call (sample names in the VCF Header line).'

File: gcp_variant_transforms/options/variant_transform_options.py [hunk 2]
```diff
@@ -219,6 +216,10 @@ def add_arguments(self, parser):
         help=('Value to use instead of null for numeric (float/int/long) lists.'
               'For instance, [0, None, 1] will become '
               '[0, `null_numeric_value_replacement`, 1].'))
+    parser.add_argument(
+        '--include_call_name',
+        type='bool', default=False, nargs='?', const=True,
+        help=('Add raw sample name as a separate column in sample record.'))
```
Comment (samanvp) [inline gcp_variant_transforms/options/variant_transform_options.py:222]: how about:
''Add raw sample name (from VCF Header line) as a subfield under the call column in addition to sample_id.''

File: gcp_variant_transforms/transforms/variant_to_avro.py [hunk 3]
```diff
@@ -56,6 +57,7 @@ def __init__(
         numeric (float/int/long) lists. For instance, [0, None, 1] will become
         [0, `null_numeric_value_replacement`, 1]. If not set, the value will set
         to bigquery_util._DEFAULT_NULL_NUMERIC_VALUE_REPLACEMENT.
+      include_call_name: If true, sample name will be included with sample ID.
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/variant_to_avro.py:60]: s/with/in addition to/

File: gcp_variant_transforms/transforms/variant_to_bigquery.py [hunk 4]
```diff
@@ -89,6 +90,7 @@ def __init__(
         numeric (float/int/long) lists. For instance, [0, None, 1] will become
         [0, `null_numeric_value_replacement`, 1]. If not set, the value will set
         to bigquery_util._DEFAULT_NULL_NUMERIC_VALUE_REPLACEMENT.
+      include_call_name: If true, sample name will be included with sample ID.
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/variant_to_bigquery.py:93]: s/with/in addition to/

**Review by samanvp — APPROVED** [review]
LGTM except for a couple of nits.

Thanks Tural!

File: gcp_variant_transforms/libs/schema_converter.py (hunk 1 above)
Comment (tneymanov) [reply gcp_variant_transforms/libs/schema_converter.py]: Done.

File: gcp_variant_transforms/transforms/variant_to_avro.py (hunk 3 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/variant_to_avro.py]: Done.

File: gcp_variant_transforms/options/variant_transform_options.py (hunk 2 above)
Comment (tneymanov) [reply gcp_variant_transforms/options/variant_transform_options.py]: Done.

File: gcp_variant_transforms/transforms/variant_to_bigquery.py (hunk 4 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/variant_to_bigquery.py]: Done.

**Review by tneymanov — COMMENTED** [review]
Thanks, Saman.
Addressed the comments.

**Pull request merged**